File: javascript/roberto.js
var fs = require('fs');

var start = new Date();

var ar = [];
var letters = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ';

for(var i = 0; i < 10000; i++) {
	ar.push(letters[Math.floor(Math.random() * letters.length)]
		+ letters[Math.floor(Math.random() * letters.length)]
		+ letters[Math.floor(Math.random() * letters.length)]
		+ letters[Math.floor(Math.random() * letters.length)]);
}

ar.sort(function (a, b) {
	return a > b ? 1 : -1;
});

var output = ar.join('\n');

fs.writeFile('./roberto_output.txt', output, function(err) {
	if(err) throw err;
	console.log('Elapsed time: ' + (new Date() - start) + 'ms\n');
});

File: csharp/roberto.cs
using System.IO;

DateTime start = DateTime.Now;

string letters = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";

List<string> ar = new List<string>();

Random rnd = new Random();
for(int i = 0; i < 10000; i++) {
	ar.Add("" + letters[rnd.Next(0, letters.Count())]
		+ letters[rnd.Next(0, letters.Count())]
		+ letters[rnd.Next(0, letters.Count())]
		+ letters[rnd.Next(0, letters.Count())]);
}

ar.Sort();

using(StreamWriter writer = new StreamWriter("roberto_output.txt")) {
	writer.Write(string.Join("\n", ar.ToArray()));
}

Console.WriteLine("Elapsed time: " + (DateTime.Now - start).TotalMilliseconds + "ms");

